modules/govuk_env_sync/files/govuk_env_sync.sh

```diff
@@ -57,6 +57,21 @@ function is_writable_mongo {
   mongo --quiet --eval "print(db.isMaster()[\"ismaster\"]);" "localhost/$database"
 }
 
+function is_writable_elasticsearch {
```

Comment:
Hi @rtrinque ,

Could you please tell us where ** is_writable_elasticsearch** is called/used.

Reply:
This is called [here](https://github.com/alphagov/govuk-puppet/pull/7863/files#diff-4bba70a7d7214e66f3873d258dfa41bcR169).